sdk/tables/azure-data-tables/samples/sample_query_table.py

```diff
@@ -80,17 +80,6 @@ def sample_query_entities(self):
                     query_filter=name_filter, select=["Brand", "Color"], parameters=parameters
                 )
 
-                for entity_chosen in queried_entities:
-                    print(entity_chosen)
-
-                print("Sample for querying entities withtout metadata:")
-                print("Entities with name: marker")
-                parameters = {"name": "marker"}
-                name_filter = "Name eq @name"
-                headers = {"Accept": "application/json;odata=nometadata"}
-                queried_entities = table_client.query_entities(
-                    query_filter=name_filter, select=["Brand", "Color"], parameters=parameters, headers=headers
-                )
```

Comment:
We have the nometadata test for `query_entities()`: https://github.com/Azure/azure-sdk-for-python/blob/ce1d00d1a08e1c5866a45417973ce483d845dedf/sdk/tables/azure-data-tables/tests/test_table_entity.py#L1498-L1512